src/frontends/paddle/src/op/strided_slice.cpp

```diff
@@ -0,0 +1,98 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+#include <limits.h>
+
+#include "default_opset.hpp"
+#include "openvino/frontend/paddle/node_context.hpp"
+
+namespace ov {
+namespace frontend {
+namespace paddle {
+namespace op {
+NamedOutputs strided_slice(const NodeContext& node) {
+    const auto data = node.get_input("Input");
+    const auto axes = node.get_attribute<std::vector<int32_t>>("axes");
+    Output<Node> start_idx_node, end_idx_node, strides_idx_node;
+    if (node.has_input("StartsTensor")) {
+        start_idx_node = node.get_input("StartsTensor");
+    } else if (node.has_input("StartsTensorList")) {
+        auto inputs = node.get_ng_inputs("StartsTensorList");
+        start_idx_node = std::make_shared<default_opset::Concat>(inputs, 0);
+    } else {
+        auto starts = node.get_attribute<std::vector<int32_t>>("starts");
+        start_idx_node = default_opset::Constant::create(element::i32, {starts.size()}, starts);
+    }
+
+    if (node.has_input("EndsTensor")) {
+        end_idx_node = node.get_input("EndsTensor");
+    } else if (node.has_input("EndsTensorList")) {
+        auto inputs = node.get_ng_inputs("EndsTensorList");
+        end_idx_node = std::make_shared<default_opset::Concat>(inputs, 0);
+    } else {
+        auto ends = node.get_attribute<std::vector<int32_t>>("ends");
+        end_idx_node = default_opset::Constant::create(element::i32, {ends.size()}, ends);
+    }
+
+    if (node.has_input("StridesTensor")) {
+        strides_idx_node = node.get_input("StridesTensor");
+    } else if (node.has_input("StridesTensorList")) {
+        auto inputs = node.get_ng_inputs("StridesTensorList");
+        strides_idx_node = std::make_shared<default_opset::Concat>(inputs, 0);
+    } else {
+        auto strides = node.get_attribute<std::vector<int32_t>>("strides");
+        strides_idx_node = default_opset::Constant::create(element::i32, {strides.size()}, strides);
+    }
+
+    const auto shape_node = std::make_shared<default_opset::ShapeOf>(data, element::Type_t::i32);
+    const auto shape_shape_node = std::make_shared<default_opset::ShapeOf>(shape_node, element::i32);
```

Comment:
nit: instead of `shape_shape_node`, call it `rank_node`.

Reply:
have changed